### ISSUE TITLE
Make one big tagline under the header

### ISSUE THREAD

**Issue by ghost** [duangish/duangish.github.io#5]
Need to reduce text by 50% into one big tagline. detextify the page

**Comment by clemsos** [comment]
Yes, we need to add some charts or viz. Business people love those

**Comment by ghost** [comment]
Yeah, but for now, focus on the big nice page, right? but chart showing usa
vs china gdp or something haha, I think this should be secondary.

**Comment by clemsos** [comment]
add the word" research" to h2 > "A global research conversation about Chinese Technology Culture." 

maybe add knowledge somewhere in h3 ?
